packages/eslint-plugin/README.md

```diff
@@ -6,13 +6,13 @@ Endo-specific plugin
 
 You'll first need to install [ESLint](http://eslint.org):
 
-```
+```sh
```

Comment:
I think the `$` prompt makes this not `sh` code. I think it's `console` or some such.

Reply:
TIL "```console". I had no idea. Thx. Done.